starlette/middleware/sessions.py

```diff
@@ -49,14 +49,17 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
 
         async def send_wrapper(message: Message) -> None:
             if message["type"] == "http.response.start":
+                # Detect ASGI root path
```

Comment:
I don't think we need this comment - it's not really adding any value here.

```suggestion
```

Reply:
Done